pallets/gear/src/tests.rs

```diff
@@ -320,13 +322,22 @@ fn delayed_send_from_reservation_not_for_mailbox() {
         run_to_next_block(None);
         assert!(Gear::is_initialized(pid));
 
+        let reservation_amount = <Test as Config>::MailboxThreshold::get();
+        let sending_delay = 1u32;
+        let sending_delay_hold_bound: HoldBound<Test> =
+            HoldBoundBuilder::new(StorageType::DispatchStash).duration(sending_delay as u64);
+        let sending_delay_gas = sending_delay_hold_bound.lock_amount();
+        // Sending delayed msg from a reservation checks: reservation_amount - delay_hold - mailbox_therhsold.
+        // Current example sets reseravtion_amount to mailbox_threshold. So, obviously, sending message
+        // from a reservation is impossible - reservation has insufficient gas reserved.
```

Comment:
Do we have test for exact gas amount of mailbox threshold + dispatch stash holding fee? Would be nice to have

Reply:
Adjusted the `delayed_reservations_to_mailbox` with a calculation for an exact reservation amount